### PR TITLE
Ops get an ion carbine instead of an ion rifle

### PR DESCRIPTION
**Pull request by Cenrus**
Unlike the other operative weapons the ion rifle is bulky, so nobody really uses it. This PR replaces it with a carbine which can fit in bags.

:cl:
tweak: Changed the ion rifle on the syndicate base to an ion carbine
/:cl:

### CODE REVIEW

File: _maps/map_files/generic/CentCom.dmm [hunk 1]
```diff
@@ -8738,12 +8738,10 @@
 /area/syndicate_mothership/control)
 "zy" = (
 /obj/structure/table,
-/obj/item/gun/energy/ionrifle{
-	pin = /obj/item/firing_pin
-	},
 /obj/effect/turf_decal/stripes/line{
 	dir = 1
 	},
+/obj/item/gun/energy/ionrifle/carbine,
```
Comment (lordScrubling) [inline _maps/map_files/generic/CentCom.dmm:8744]: i think it needs a firing pin

Comment (BrumalLake) [inline _maps/map_files/generic/CentCom.dmm]:
```suggestion
/obj/item/gun/energy/ionrifle/carbine{
	pin = /obj/item/firing_pin
},
```

Comment (PowerfulBacon) [reply _maps/map_files/generic/CentCom.dmm]: needs a tab before the },

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Ion carbine doesnt come with a firing pin.

**Comment by Cenrus** [comment]
added a pinned subtype for admin or mapping use

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**